sdbuild/packages/python_packages_bionic/qemu.sh

```diff
@@ -48,6 +48,5 @@ psutil==5.7.0
 pybind11==2.5.0
 EOT
 
-python3.6 -m pip install -U pip
```

Comment:
I disagree with removing the line entirely, this would result in a build that is inconsistent with the current 2.6

I'd suggest we do 
```python
python3.6 -m pip install -U pip==20.2.3
```